src/web/pages.rkt

```diff
@@ -42,7 +42,8 @@
      (make-points-json result out ctx)]))
 
 (define (get-interactive-js result ctx)
-  (match-define (job-result _ _ _ _ _ (improve-result _ _ start _ end)) result)
+  (match-define (job-result _ _ _ _ _ 
+                 (improve-result _ _ start _ end _)) result)
```

Comment:
I know it's not your code, but I would probably do two things here.

1. Since `get-interactive-js` should only run on successes (when the `improve-result` is available), I would put `'success` in the `job-result` `status` field. That makes the invariant cleaner.
2. I would match once to get the `backend` and again to the get the `start` and `end`, just putting them on different lines.